example/svm_mnist/svm_mnist.py

```diff
@@ -41,18 +44,24 @@
 fc3  = mx.symbol.FullyConnected(data = act2, name='fc3', num_hidden=10)
 
 # Here we add the ultimate layer based on L2-SVM objective
-mlp = mx.symbol.SVMOutput(data=fc3, name='svm')
+mlp_svm_l2 = mx.symbol.SVMOutput(data=fc3, name='svm_l2')
+
+# With L1-SVM objective
+mlp_svm_l1 = mx.symbol.SVMOutput(data=fc3, name='svm_l1', use_linear=True)
 
-# To use L1-SVM objective, comment the line above and uncomment the line below
-# mlp = mx.symbol.SVMOutput(data=fc3, name='svm', use_linear=True)
+# Compare with softmax cross entropy loss
+mlp_softmax = mx.symbol.SoftmaxOutput(data=act2, name='softmax')
```

Comment:
`fc3` instead of `act2` for the softmaxOutput layer?

Reply:
good catch